### DISCUSSION TITLE
Quasar CLI Update Notice

### DISCUSSION THREAD

**Discussion by susansiow** [quasarframework/quasar#11873] [CLI - PWA mode] [answered]
When I run `quasar dev` the following message appears.

<img width="247" alt="quasar" src="https://user-images.githubusercontent.com/53169547/147903446-9229b83d-cdc6-49c0-a042-c4953a6ab930.png">

Questions:

1. Is there a reference on what will be affected with the CLI upgrade?
2. Upgrading the CLI means the Quasar's version will be upgraded at the same time? If yes, how can I keep Quasar version 1 and not upgrade to version 2 while upgrading the CLI?

**Comment by MilosPaunovic** [comment, marked as answer]
1. You could check [Release notes](https://quasar.dev/start/release-notes) section of the documentation, focus primarily on the `@quasar/cli` tab.
2. Nope, it does not affect Quasar itself.
